app/models/acl_dao.rb

```diff
@@ -0,0 +1,91 @@
+class AclDao
+  attr_accessor :access_token, :client_id, :base_url
+
+  def initialize(token, client_id, base_url)
+    @access_token = token
+    @client_id = client_id
+    @base_url = base_url
+  end
+
+  def get_role(user_id)
+
+    # Sets whether the user is a specific role type
+    # Note: user could be all three.
+    user_is_daac_curator = false
+    user_is_arc_curator = true
+    user_is_admin = false
+
+    results = search_acls_by_user(user_id, 1)
+    noPages = (results['hits']/2000).ceil
+    items = results['items']
```

Comment:
I think you should use the `Array.wrap` here, and remove it from line 25

Reply:
Agree makes sense.